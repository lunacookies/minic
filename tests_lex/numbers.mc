123
456
999999999
0
