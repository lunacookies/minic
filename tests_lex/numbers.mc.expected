{
	NUMBER 0..3
	NUMBER 4..7
	NUMBER 8..17
	NUMBER 18..19
}
